### DISCUSSION TITLE
Error: creating EC2 Subnet: InvalidSubnet.Conflict: The CIDR '10.0.1.0/24' conflicts with another subnet

### DISCUSSION THREAD

**Discussion by gmonika136** [learndevopsonline/learndevopsonline#1512] [Q&A]
Hi Raghu,

When i am creating to 4 subnets in aws, but getting the below error.

![Capture](https://github.com/learndevopsonline/learndevopsonline/assets/129938676/20cd2a84-e1f4-4b4c-9476-14e242b1c6d8)

**Comment by sreedharm07** [comment]
Hi @gmonika136 
here you have given same cidr range  please give different cidr 
instead of this ==["10.0.0.0/24", "10.0.1.0/24"]  give ["10.0.2.0/24", "10.0.3.0/24"]

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/6ba0fb37-9360-45b1-92f3-1567e7b1b40c)
